app/assets/javascripts/phrasing.js

```diff
@@ -135,6 +135,7 @@ var phrasing_setup = function(){
         }
       },
       error: function(e){
+        console.log("Phrasing:", e.responseText);
```

Comment:
Remove console.log

Reply:
You sure?
I think it's good to have an error log cause only admins of pages will be able to cause it. And since it might be triggered by a client admin, he can be instructed to investigate the console and send the error to the developers.